core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -314,7 +314,6 @@ class GroupMetadataManager(brokerId: Int,
 
       case None =>
         responseCallback(Errors.NOT_COORDINATOR)
-        None
```

Comment:
Just for my understanding since my scala is a bit rusty. `None` is useless here because the return type of the method is `Unit`, right?

Reply:
Exactly. This is how ide (IntelliJ 2021.1.2 + scala 2021.1.21) is showing - a dead code with a gray marking.

![Screenshot from 2021-06-08 22-56-14](https://user-images.githubusercontent.com/2375128/121197999-be71ef00-c8ac-11eb-981b-25bbc8e87404.png)